openedx/core/djangoapps/schedules/management/commands/tests/test_send_recurring_nudge.py

```diff
@@ -117,6 +118,27 @@ def test_no_course_overview(self, mock_schedule_send):
         # is null.
         self.assertEqual(mock_schedule_send.apply_async.call_count, 0)
 
+    @patch.object(tasks, '_recurring_nudge_schedule_send')
+    def test_send_after_course_end(self, mock_schedule_send):
+        user1 = UserFactory.create(id=tasks.RECURRING_NUDGE_NUM_BINS)
+
+        schedule = ScheduleFactory.create(
+            start=datetime.datetime(2017, 8, 3, 20, 34, 30, tzinfo=pytz.UTC),
+            enrollment__user=user1,
+        )
+        schedule.enrollment.course = CourseOverviewFactory()
+        schedule.enrollment.course.end = datetime.datetime.now(pytz.UTC) - datetime.timedelta(days=1)
+
+        test_time = datetime.datetime(2017, 8, 3, 20, tzinfo=pytz.UTC)
```

Comment:
We should conventionalize what we call the datetime used by the daily nudge query. It was called "target_hour" in tasks.py when we queried by the hour. Once you introduced binning, I changed it to "target_day" in tasks and in tests. I see that we use "target_time" and "target_date" too. What do you think is the right name?

Reply:
I guess it's pretty confusing that I keep changing it up. I'm trying to standardize on using `*_date` to mean a datetime object with time and `*_day` to mean a datetime object with truncated (zeroed-out) time. I forgot to change the tests, so I'd want to make this `test_date`. `test_time` makes me think it's a time object with no date information. Maybe day and date are not the best words for it since they are pretty much synonyms, so I'm open to other suggestions.

Reply:
`*_datetime` and `*_day`?